docs/release-notes/stable versions.md

```diff
@@ -0,0 +1,31 @@
+---
+id: stable-versions
+title: Mia-Platform stable versions
+sidebar_label: Stable versions
+---
+
+A **stable release** is a version that has been thoroughly tested and is therefore as reliable as possible. It does not include all the latest new features of a beta release.
```

Comment:
I would not mention beta release here. Where is it defined? I think we do not talk of beta release anywhere, so I would not introduce it here unless we decide to call other versions as beta versions